### PR TITLE
docs: `LowessRegression` usage docstring

### PR DESCRIPTION
**Pull request by david26694**
Hey @FBruzzesi , adding a docstring for LowessRegression. A change has been done in a test file automatically by ruff, let me know if you want me to revert it.

# Description

Fixes https://github.com/koaning/scikit-lego/issues/650

## Type of change

Docs

## Checklist:

- [x] My code follows the style guidelines (ruff)
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (also to the readme.md)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added tests to check whether the new feature adheres to the sklearn convention
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Review by koaning — APPROVED** [review]
Ah nice, good to have. Thanks!

**Comment by koaning** [comment]
I am running the tests to be safe, but this can be merged once it all looks green.

File: sklego/linear_model.py [hunk 1]
```diff
@@ -49,6 +49,25 @@ class LowessRegression(BaseEstimator, RegressorMixin):
         The training data.
     y_ : np.ndarray of shape (n_samples,)
         The target (training) values.
+
+
+    Example
```
Comment (FBruzzesi) [inline sklego/linear_model.py:54]: A tiny one, but if I remember correctly, rendering will differ

```suggestion
    Examples
```

File: tests/test_meta/test_zero_inflated_regressor.py [hunk 2]
```diff
@@ -112,8 +112,8 @@ def test_score_samples():
     # Where the classifier prediction is zero, then the score is by something greater than 0.
     assert approx_gte(scores[~pred_is_non_zero], preds[~pred_is_non_zero])
 
-def test_no_predict_proba():
 
+def test_no_predict_proba():
```
Comment (FBruzzesi) [inline tests/test_meta/test_zero_inflated_regressor.py:116]: No worries about this

**Review by FBruzzesi — COMMENTED** [review]
Thanks for the PR. I am late to the party, but left a comment :)

**Pull request merged**